GoogleUtilities.podspec

```diff
@@ -0,0 +1,65 @@
+Pod::Spec.new do |s|
+  s.name             = 'GoogleUtilities'
+  s.version          = '5.0.4'
+  s.summary          = 'Firebase Utilities for iOS (plus community support for macOS and tvOS)'
+
+  s.description      = <<-DESC
+Firebase Utilities including Network, Environment, Logger, and Swizzling.
```

Comment:
Do we want to have a note here saying not for public use, or "Internal Google Utilities for iOS including Network, Environment, Logger, and Swizzling"